src/Integration/NewConnectedMode/ConfigurationProvider.cs

```diff
@@ -21,27 +21,26 @@
 using System;
 using System.Diagnostics;
 using SonarLint.VisualStudio.Integration.Persistence;
-using SonarLint.VisualStudio.Integration.Resources;
 
 namespace SonarLint.VisualStudio.Integration.NewConnectedMode
 {
     internal class ConfigurationProvider : IConfigurationProvider
     {
         private readonly ISolutionBindingSerializer legacySerializer;
-        private readonly ISolutionBindingSerializer newConnectedModeSerializer;
+        private readonly IConfigurationProvider wrappedConfigProvider;
 
-        public ConfigurationProvider(ISolutionBindingSerializer legacySerializer, ISolutionBindingSerializer newConnectedModeSerializer)
+        public ConfigurationProvider(ISolutionBindingSerializer legacySerializer, IConfigurationProvider wrappedConfigProvider)
```

Comment:
[minor][cosmetic] I know that `config` is a known abbreviation but I can't say I am a big fan of them.